### PR TITLE
Creating models

### PR DESCRIPTION
**Pull request by g-palanskiy**

### CODE REVIEW

File: app/Http/Controllers/PostController.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+namespace App\Http\Controllers;
+
+use App\Models\Post;
+use Illuminate\Http\Request;
+
+class PostController extends Controller
+{
+    /**
+     * Display a listing of the resource.
```
Comment (ekaterinagaltsova) [inline app/Http/Controllers/PostController.php:11]: remove unnecessary

File: database/migrations/2023_01_23_064630_create_posts_table.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+      $table->dropForeign('posts_user_id_foreign');
```
Comment (ekaterinagaltsova) [inline database/migrations/2023_01_23_064630_create_posts_table.php]: по аналогии 
 Schema::table('posts', function (Blueprint $table) {
            $table->dropForeign(['user_id']);
        });
        
        ?

File: database/migrations/2023_01_23_064630_create_posts_table.php [hunk 3]
```diff
@@ -15,7 +15,7 @@ public function up()
     {
       Schema::create('posts', function (Blueprint $table) {
         $table->id();
-        $table->integer('user_id');
+        $table->bigInteger('user_id');
```
Comment (ekaterinagaltsova) [inline database/migrations/2023_01_23_064630_create_posts_table.php:18]: $table->bigInteger('user_id');  -> redundant line
The foreignId method creates an UNSIGNED BIGINT equivalent column

File: database/migrations/2023_01_23_064630_create_posts_table.php [hunk 4]
```diff
@@ -15,7 +15,7 @@ public function up()
...
         $table->foreign('user_id')->references('id')->on('users')->onDelete('cascade');
```
Comment (ekaterinagaltsova) [inline database/migrations/2023_01_23_064630_create_posts_table.php:19]: Laravel 9 syntax?
$table->foreignId('user_id')->constrained()->onDelete('cascade');

**Review by D-pavlo-v — APPROVED** [review]

**Pull request merged**